reflex-dom-core/src/Reflex/Dom/Widget/Basic.hs

```diff
@@ -144,7 +144,7 @@ button t = do
 
 --TODO: Should this be renamed to 'widgetView' for consistency with 'widgetHold'?
 -- | Given a Dynamic of widget-creating actions, create a widget that is recreated whenever the Dynamic updates.
---   The returned Event of widget results occurs when the Dynamic does.
```

Comment:
Shouldn't this say widget of Event instead?

Reply:
Event of (widget results)

Reply:
But I agree it's a little hard to parse. Perhaps:

The returned Event occurs whenever the child widget is updated, which is at post-build in addition to the times at which the input Dynamic is updated, and its value is the result of running the widget.